vue/components/ui/atoms/dropdown/dropdown.vue

```diff
@@ -148,6 +148,15 @@
 .dropdown-container .dropdown > .dropdown-item.selected,
 .dropdown-container .dropdown > .dropdown-item.highlighted {
     background-color: $soft-blue;
+    font-weight: 700;
+}
+
+.dropdown-container .dropdown > .dropdown-item.disabled {
+    background-color: transparent;
+    color: #9da5af;
+    cursor: not-allowed;
+    font-style: normal;
+    font-weight: 200;

```

Comment:
I don't think you need to change the font weight one disabled, it does not match the design:
<img width="335" alt="imagem" src="https://user-images.githubusercontent.com/25725586/156158522-97e4e568-65b8-4378-8fb0-1c84d50fcbe6.png">

Following the design, it's just changing the opacity of item:
<img width="774" alt="imagem" src="https://user-images.githubusercontent.com/25725586/156160837-c0f04815-ba9a-4329-abf9-63eaf7ba5599.png">

```suggestion
.dropdown-container .dropdown > .dropdown-item.disabled {
    opacity: 0.5;
    cursor: not-allowed;
```